core/services/vrf/listener_v2.go

```diff
@@ -340,11 +328,19 @@ func (lsn *listenerV2) processRequestsPerSub(
 
 	// Attempt to process every request, break if we run out of balance
 	for _, req := range reqs {
+		fromAddress, err := lsn.gethks.GetRoundRobinAddress(lsn.fromAddresses()...)
+		if err != nil {
+			lggr.Errorw("Couldn't get next from address", "error", err)
```

Comment:
Nit: I think the logger field we're using for error has been `"err"` rather than `"error"`.

Reply:
Fixed